includes/User/Capabilities.php

```diff
@@ -36,7 +38,7 @@
  *
  * @package Google\Web_Stories\User
  */
-class Capabilities implements Activateable, Deactivateable {
+class Capabilities implements Activateable, Deactivateable, Initialize_Site, Delete_Site {
```

Comment:
I don't think this will have any effect, because `Capabilities` is not a service, and these interfaces are all meant to be used on services.

Since it`s not a service, there's no way the plugin can know it needs to instantiate it.

Reply:
Yeah, I was looking into this last night. You are right as the unit tests failed. It is a nice pattern, so I would like to use it.

Reply:
You should be able to just turn `Capabilities` into a `Service` without much work. Just make sure you check all the places where you're using it to ensure it still works, and keep in mind that it will be shared by default if retrieved through the `Services` container, so it might make sense to just build it as a shared service right away.